lib/internal/perf/observe.js

```diff
@@ -307,7 +306,9 @@ class PerformanceObserver {
       maybeIncrementObserverCount(type);
       if (buffered) {
         const entries = filterBufferMapByNameAndType(undefined, type);
-        ArrayPrototypePushApply(this.#buffer, entries);
+        for (let i = 0; i < entries.length; i++) {
+          ArrayPrototypePush(this.#buffer, entries[i]);
+        }
```

Comment:
I understand the necessary fix but we should benchmark this to make sure it's not too much of a regression. Might make sense to split `entries` up and still use `...PushApply` with smaller, safer chunks